labman/db/process.py

```diff
@@ -382,14 +419,16 @@ def create(cls, user, primer_set, master_set_order, creation_date=None):
             instance = cls(TRN.execute_fetchlast())
 
             creation_date = instance.date
-            plate_name_suffix = creation_date.strftime('%Y-%m-%d')
+            plate_name_suffix = creation_date.strftime(
+                Process.get_date_format())
             primer_set_plates = primer_set.plates
             check_name = '%s %s' % (primer_set_plates[0].external_id,
                                     plate_name_suffix)
             if plate_module.Plate.external_id_exists(check_name):
                 # The likelihood of this happening in the real system is really
                 # low, but better be safe than sorry
-                plate_name_suffix = datetime.now().strftime('%Y-%m-%d %H:%M')
+                plate_name_suffix = "{0} {1}".format(plate_name_suffix,
+                                                     randrange(1000, 9999))
```

Comment:
Is this still necessary, given that we're already including the `%H:%M` in the suffix?